### ISSUE TITLE
Windows 11

### ISSUE THREAD

**Issue by Sprite9268** [Nibre/MotherVR#232]
When I install the mod I can't open the game for steam. When will you be fixing this?

**Comment by Mdmanga** [comment]
Yes, please could it be possible to fix this problem. Is it complicated under Win 11 ?
I hope not. I really want to try this game in VR, it's the only reason I bought it...
Thank in advance for developpers, you did a good job and I'm sure you will find a solution.

**Comment by Mdmanga** [comment]
Any news guys ?

**Comment by J4cKP0t777** [comment]
...sigh

**Comment by J4cKP0t777** [comment]
u rekon problem is just with the steam version?  its on sale at epic atm

**Comment by Mdmanga** [comment]
Someone said it works on win 11 with EPic game version. But I bought it on steam so I can't transfer it to steam ?
How can I do this ?

**Comment by J4cKP0t777** [comment]
i have it working on Epic store version with WIn11.Working well too. 
 I got a refund from Steam and then just bought the Epic version :)

**Comment by Mdmanga** [comment]
Impoosible for me to have a refund, they refused. I don't want to buy it twice. There may be a reason why it can work with Epic game version but not with steam.

**Comment by J4cKP0t777** [comment]
i havnt tested but sometimes this works... 
To run the game from Steam with SteamVR, disable the Steam VR Theater.
1. Right click the game in Steam and select 'Properties'.
2. Uncheck 'Use Desktop Game Theatre with SteamVR.'

otherwise maybe consider rebuying from epic while its on sale? I would recommend the game and the mod works well but understand your reluctance to pay twice.

**Comment by Sprite9268** [comment]
> 

i disabled game theatre but still wont work and I cant get a refund for steam

**Comment by Mdmanga** [comment]
Me neither, I can't get the refund and I don't want to buy this game twice. PLease developpers, I'm sure the solution is quite simple for you. It works with epic game version and Win11 bit not with Steam.
Thank you

**Comment by renegadespork** [comment]
This issue was already reported back on September 1st, 2021: https://github.com/Nibre/MotherVR/issues/229

**Comment by Pocokk** [comment]
I love how instead of the developer come back to the mod and fix it, everyone tries to create these arbitrary solutions for this problem.

**Comment by hiabex** [comment]
Hi, I've recently downloaded the Epic Games version of Alien Isolation, however, adding the mod dll to the appropriate folder causes the game to hang on first boot, thus making it impossible to activate the mod in the game menu. 
If I remove the dll, the game boots up no problem in 2D mode. Obviously as this thread is Win11 related, that's  my OS. 
Has anyone here run into my issue? If yes, have you found a work around? I'm dying to try Isolation in VR, so this non-starter is turning out to be a massive frustration.
